src/Tools/ExternalAccess/OmniSharp/Rename/OmniSharpRenamer.cs

```diff
@@ -9,12 +9,13 @@
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.CodeCleanup;
 using Microsoft.CodeAnalysis.Rename;
+using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp
 {
     internal static class OmniSharpRenamer
     {
-        public readonly record struct RenameResult(Solution Solution, string? ErrorMessage);
+        public readonly record struct RenameResult(Solution? Solution, string? ErrorMessage);
```

Comment:
Tag @JoeRobich and @333fred to check here

Reply:
So the basic problem is originally OmniSharp is supposed to the result would always come with a Solution.
And but if the rename accept a symbol not in source https://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/Rename/ConflictEngine/ConflictResolver.cs,94
It would return a string.
And because we suppress the nullable warning so the Solution here could be null.

Reply:
This looks fine.